netmanager/src/utils/dateTime.js

```diff
@@ -2,14 +2,8 @@ import moment from "moment-timezone";
 
 export const formatDateString = (
   t,
-  format = "YYYY-MM-DD HH:mm",
-  tz = "Africa/Kampala"
-) => {
-  return moment
-    .utc(t, format)
-    .tz(tz || "Africa/Kampala")
```

Comment:
The timezone function (`tz()`) was removed because it purpose is basically already achieved with the `.utc()` and `.format()` function combination. It was the root cause of the bug.